src/main/java/interface_adapter/sub_menu/SubViewController.java

```diff
@@ -1,5 +1,21 @@
 // Author: Diego
 package interface_adapter.sub_menu;
 
+
+import use_case.sub_menu.SubViewInputBoundary;
+import use_case.sub_menu.SubViewInputData;
+
+import java.util.Objects;
+
 public class SubViewController {
+
+    final SubViewInputBoundary userSubViewUseCaseInteractor;
+    public SubViewController(SubViewInputBoundary userSubViewUseCaseInteractor) {
+        this.userSubViewUseCaseInteractor = userSubViewUseCaseInteractor;
+    }
+
+    public void execute(String next_panel) {
+        SubViewInputData subViewInputData = new SubViewInputData(next_panel);
+        userSubViewUseCaseInteractor.execute(subViewInputData);
```

Comment:
Interesting way of implementing the execute method. Just be aware that if submenu needs to do anything besides changing the current view, the execute method would not work. If things need to change, maybe consider renaming execute to "switchView" or something else more descriptive of it's purpose!